Source/ARTChannels.h

```diff
@@ -18,6 +18,6 @@
 - (ChannelType)get:(NSString *)name;
 - (ChannelType)get:(NSString *)name options:(ARTChannelOptions *)options;
 - (void)release:(NSString *)name;
-- (id<NSFastEnumeration>)iterate;
+- (id<NSFastEnumeration>)copyIntoIteratorWithMapper:(id (^)(ChannelType))mapper;
```

Comment:
I'm not clear whether this is a user API. Is this a breaking change?

Reply:
No, it isn't. ARTChannels is used internally by ARTRealtimeChannelsInternal and ARTRestChannelsInternal. The exposed method is still `iterate`.

Reply:
Thanks for explaining.